medical/README.rst

```diff
@@ -6,82 +6,64 @@
 Odoo Medical
 ============
 
-What is Odoo Medical
---------------------
+This module extends Odoo with the base functionality of medical patients, centers, and doctors.
 
-**Odoo Medical** is a multi-user, highly scalable, centralized Electronic Medical
-Record (EMR) and Hospital Information System for odoo.
+**Some patient attributes available:**
```

Comment:
Reduced to "This module extends Odoo with the base functionality of medical patients and centers." Removed attribute descriptions.